drivers/SmartThings/samsung-audio/src/command.lua

```diff
@@ -388,22 +367,7 @@ function Command.play_streaming_uri(ip, uri)
   log.trace("Triggering UPnP Command Request for [Audio Notification -> SetUrlPlayback]")
   local response_map = nil
   if ip then
-    local path = format_streaming_path(uri)
-    local url = format_url(ip, path)
-    log.trace(string.format("Final Notification Command URL for making Audio Notification http request = %s", url))
-    response_map = handle_http_request(ip, url)
-    --- Adding extra debug logs in case some issue comes in future
-    log.debug("Response Map table with https: ", utils.stringify_table(response_map))
-    local resp = get_resp(response_map)
-    log.debug("resp method and error code : ", utils.stringify_table(resp))
-    if resp.method == "ErrorEvent" then
-      log.info(string.format("Recieved error code %s",resp.err_code))
-      response_map= fallback_to_http(ip,uri)
-    end
-    if resp.method == "PausePlaybackEvent" then
-      log.info(string.format("Recieved error code %s",resp.err_code))
-      response_map= fallback_to_http(ip,uri)
-    end
+    response_map = fallback_to_http(ip, uri)
```

Comment:
Should we remove the function and just format the URI here? Since we are not directly calling ```handle_http_request()``` directly anyway.

Reply:
I kept in order to maintain the understanding of how we reached to the conclusion of falling back to http. It is just for understanding.